schema/Electrometer.yaml

```diff
@@ -68,7 +68,7 @@ properties:
   timeout:
     description: The timeout of the serial connection.
     type: integer
-    default: 1
+    default: 2
   xonxoff:
```

Comment:
Is this needed? I don't see it used anywhere in the code. please delete it if you can.